src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -68,20 +89,30 @@ const AuditLogsTable = ({ showModel = false }: Props) => {
     (column) => showModel || column.accessor !== "model"
   );
 
-  const [, setQueryParams] = useQueryParams<{ page: string | null }>({
+  const [limit, setLimit] = useState<number>(DEFAULT_LIMIT_VALUE);
+  const [{ page }, setQueryParams] = useQueryParams<{ page: string | null }>({
     page: null,
```

Comment:
You can set the default here to `"1"` instead of checking for null when setting the offset.